ontology/yaml/resources/HVAC/entity_types/ASHP.yaml

```diff
@@ -44,4 +44,22 @@ ASHP_SS_SHWTC_SCHWTC_HWRWISOVM_CHWRWISOVM_HCPC2X_CHCPC2X:
   - HWRWISOVM
   - CHWRWISOVM
   - HCPC2X
-  - CHCPC2X
\ No newline at end of file
+  - CHCPC2X
+
+ASHP_SS_CHWRISOVPM_CHWDT_CHDX2SC_CFVSC_RVINJM2X_CPC2X_CHWDPC_HWDPM_SHWTC_HWRWISOVPM:
+  guid: "d14e8ca6-9f28-45a0-9d16-97fe9c9b4c72"
+  description: "Air source heat pump with return side isolation valve percentage control for chill and heat water, temperature differential accross chill water, refrigerant vapor injection monitoring, differential pressure monitoring for heating water, and supply heating water temperature control."
```

Comment:
Please change this so that it always says "chilled water" rather than "chill water" and "heating water" rather than "heat water: